Advanced Grid Properties: Add missing factors to grid content width calc (#30971)
TheOdinProject/curriculum: intermediate_html_css/grid/advanced_grid_properties.md

Notice how when we resize, the columns automatically know how many will fit across. If you don't think *that's* cool, you better check your pulse!

> So what's going on here specifically with `repeat(auto-fit, minmax(150px, 1fr));`? Remember that `auto-fit` will return the **highest positive integer** without overflowing the grid. So first, the browser has to know the grid's content width: In this case, it's the window's width minus margins (which we didn't explicitly set), and then minus the grid's own paddings and borders. For the sake of this example, let's pretend like our window is currently `500px` wide. Second, the browser needs to know how many grid column tracks could possibly fit in that width. To do this, it uses the minimum value in our `minmax()` function, since that will yield the highest number of items, which is `150px`. If our window is `500px` wide, this means our grid will render 3 columns. But wait, there's more! Once the browser has determined how many columns we can fit, it then resizes our columns up to the maximum value allowed by our `minmax()` function. In this case, our max size is `1fr`, so all three columns will be given an equal allotment of the space available. As we resize our window, these calculations happen in realtime and the result is what you see in the above example!

Reviewer: I'm not entirely sure why GitHub is having so much trouble showing a reasonable diff for this but oh well. Here's a more reasonable diff: https://www.diffchecker.com/ysTvQrPJ/

After a re-read, what do you think about the following suggestion? Splitting the paragraph and streamlining the phrasing a touch.
```suggestion
So what's going on here specifically with `repeat(auto-fit, minmax(150px, 1fr));`? Remember that `auto-fit` will return the **highest positive integer** without overflowing the grid. So first, the browser has to know the width of the grid's content box (i.e. ignoring margins, borders or padding).

For the sake of this example, let's pretend like our grid's content box is currently `500px` wide. Second, the browser needs to know how many grid column tracks could possibly fit in that width. To do this, it uses the minimum value in our `minmax()` function, since that will yield the highest number of items, which is `150px`. If our window is `500px` wide, this means our grid will render 3 columns. But wait, there's more! Once the browser has determined how many columns we can fit, it then resizes our columns up to the maximum value allowed by our `minmax()` function. In this case, our max size is `1fr`, so all three columns will be given an equal allotment of the space available. As we resize our window, these calculations happen in realtime and the result is what you see in the above example!
```
